TypeScript/ownerOnly.ts

```diff
@@ -18,11 +18,14 @@
  */
 
 import { CommandType, CommandControlPlugin, controller } from "@sern/handler";
-export function ownerOnly(ownerIDs: string[]) {
+export function ownerOnly(ownerIDs: string | string[], response?: string) {
```

Comment:
Old ownerOnly passed 0 parameters into the function. Is there a way to simulate this behavior with the new 2.0?
ie: 
![image](https://user-images.githubusercontent.com/76754747/221643933-414469c3-ef50-494c-bc3c-89bd767349d5.png)

Reply:
I think it would be possible but not everyone loads the ownerIds the same way so that may lead to more problems. I can look at some different solutions for this, but it may take some time as I should go to sleep soon.

Reply:
no rush!